lib/fastlane_core/configuration/configuration_file.rb

```diff
@@ -32,10 +32,13 @@ def initialize(config, path, block_for_missing)
     def method_missing(method_sym, *arguments, &block)
       # First, check if the key is actually available
       if self.config.all_keys.include? method_sym
-        value = arguments.first || (block.call if block_given?) # this is either a block or a value
-        if value
```

Comment:
That was the second half of the original bug. :smile:

The first half was that `arguments.first` would be `false` so it would try the second half the of `||` but fail the `block_given?`, resulting in `value = nil`